src/Symfony/Component/JsonEncoder/Attribute/Encodable.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonEncoder\Attribute;
+
+/**
+ * Mark a class as encodable.
+ *
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ *
+ * @experimental
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class Encodable
```

Comment:
```suggestion
final class JsonEncodable
```
to make it super clear/explicit?

Reply:
:100: